nix/mkDevShell.nix

```diff
@@ -46,9 +51,15 @@ let
       ];
     shellHook = ''
       bin=$(pwd)/tmp/bin
-      fetch-bky-as.sh $bin ${goos} ${goarch}
+      fetch-bky-as.sh ${version} $bin ${goos} ${goarch}
```

Comment:
FWIW, I would probably order the arguments as:
```
fetch-bky-as.sh $bin ${version} ${goos} ${goarch}
```
so that the arguments that  define the thing to fetch are all together.